### PR TITLE
Add WEBGL_multi_draw tests using SharedArrayBuffer.

### PR DESCRIPTION
**Pull request by kenrussell**
These are likely to require additional browser command-line arguments
since otherwise this functionality requires special configuration of
the web server hosting these tests.

Regression test for #3286.

Associated with http://crbug.com/1211185 .

### CODE REVIEW

**Comment by kenrussell** [comment]
@shrekshao : please review.

@jdashg , others: any comments welcome.

**Comment by kenrussell** [comment]
Note that the duplication slightly increases the run time of this test, but even with this addition, it's still around 2.2 seconds on my work laptop.

**Review by shrekshao — APPROVED** [review]
LGTM.
(The reason for only testing STATIC_DRAW with SharedArrayBuffer is?)

**Comment by kenrussell** [comment]
> LGTM.
> (The reason for only testing STATIC_DRAW with SharedArrayBuffer is?)

To reduce the overhead and duplication in the test.

**Comment by kenrussell** [comment]
Thanks for your review @shrekshao . I'll wait until tomorrow to merge this in case Jeff has feedback.

**Comment by kenrussell** [comment]
Merging now.

**Pull request merged**
